src/backend/distributed/operations/shard_transfer.c

```diff
@@ -397,7 +416,7 @@ citus_move_shard_placement(PG_FUNCTION_ARGS)
 	}
 
 	/* since this is move operation, we remove shards from source node after copy */
```

Comment:
comment seems outdated
```suggestion
	/* since this is move operation, we remove the placements from the metadata for the source node after copy */
```